### ISSUE TITLE
HA 1.110.x BETA shows deprecation warning.

### ISSUE THREAD

**Issue by dreimer1986** [roberodin/ha-samsungtv-custom#44]
```
[homeassistant.components.media_player] MediaPlayerDevice is deprecated, modify AlexaClient to extend MediaPlayerEntity
```

Fix is easy. As the Pull request states https://github.com/home-assistant/core/pull/34592 a simple rename is enough to fix it. Tested and proven right.

**Comment by Sjeff** [comment]
I receive the following warnings :
```
WARNING (MainThread) [homeassistant.components.media_player] MediaPlayerDevice is deprecated, modify SamsungTVDevice to extend MediaPlayerEntity
WARNING (MainThread) [homeassistant.components.media_player] MediaPlayerDevice is deprecated, modify SamsungTVDeviceQLED to extend MediaPlayerEntity
WARNING (MainThread) [homeassistant.components.media_player] MediaPlayerDevice is deprecated, modify SamsungTVDeviceWS to extend MediaPlayerEntity
```

**Comment by dreimer1986** [comment]
Yeah, sry copy paste error. That one was meant for another addin ^^ I meant the ones you pasted too

**Comment by a-r-j-a-n** [comment]
Fixed in #46 

Released in 3.0.4
